src/ert/shared/share/ert/forward-models/res/script/ecl_run.py

```diff
@@ -312,11 +312,11 @@ def _get_log_name(self, eclrun_config=None):
             self.sim.version if eclrun_config is None else eclrun_config.version
         )
 
-        return (
-            f"{self.base_name}.OUT"
-            if version.parse(eclipse_version) >= version.parse("2019.3")
-            else f"{self.base_name}.LOG"
-        )
+        logname_extension = "OUT"
+        with suppress(version.InvalidVersion):
```

Comment:
How can we have this triggered? :thinking:

Reply:
the version string for flow can be e.g. `daily` which will trigger such an exception.